### ISSUE TITLE
FITS processing is not starting

### ISSUE THREAD

**Issue by astromiki** [BHTOM-Team/bhtom2#141]
For some reason FITS processing is not starting (FITS are not being sent to CCDPhot). It happens from time to time. Example attached.
![image](https://github.com/BHTOM-Team/bhtom2/assets/19734606/bc6535c0-62df-4d55-9f15-8d6c7b056c99)

**Comment by majkelx** [comment]
Jak ten `id=157296` ?

**Comment by majkelx** [comment]
Ja widzę że już jest przetworzony, może czekał w kolejce.

**Comment by astromiki** [comment]
Tak, przetworzył się. Dość długo wisiał choć było dużo wolnych procesów. Czasem się tak dzieje, ze proces nie startuje dopóki nie skończy się przetwarzać jakiś inny. Trochę to wygląda jakby kolejka nie była na bieżąco odświeżana.

**Comment by majkelx** [comment]
To dosyć dziwne indeed, zwłaszcza że `id=157296` ma `priority=3` czyli wysoki w porównaniu z sąsiadami. priority idzie wprost do dramatique : niższa wartość wyższy priorytet i zadanie powinno być przydzielone do wykonania zgodnie z priorytetami. 

@tmac-rvx możesz zobaczyć czy mamy jakieś logi z `ccdphotsvc`, bym w nich trochę pogrzebał w wolnym czasie, ewentualnie dodał jakieś logowanie żeby lepiej rozumieć przydzielanie zadań przez `dramatiq`.
